pages/configuration/file_location.rst

```diff
@@ -137,65 +141,3 @@ MongoDB
 | Log files     | ``/var/log/mongodb/`` |
 +---------------+-----------------------+
 
-
-Omnibus package
-===============
-
-This paragraph covers Graylog installations via OVA, on AWS (via AMI), and on OpenStack using the `Graylog Omnibus package <https://github.com/Graylog2/omnibus-graylog2#readme>`_.
-
-.. _omnibus-graylog:
```

Comment:
did you checked if the `:refs:` that are deleted are used somewhere?

Reply:
Refs should be checked by the travis build (I saw the build fail yesterday when I did a change), but I can manually look for them 👍